ci/ci.sh

```diff
@@ -308,7 +308,7 @@ build_sphinx_docs() {
     if [ "${OSTYPE}" = msys ]; then
       echo "WARNING: Documentation not built on Windows due to currently-unresolved issues"
     else
-      FAST=True make html
+      FAST=True make develop
```

Comment:
this is slightly risky, will revert this after N/N for this migration. the two plugins in question currently clash and throw a warning.